src/spryker/docker/image/console/root/lib/task/spryker/init.sh

```diff
@@ -4,6 +4,5 @@ function task_spryker_init()
 {
     run "vendor/bin/console propel:install -o"
     passthru "vendor/bin/install -r docker -s queue-flush"
-    passthru "PGPASSWORD=$DB_PASS vendor/bin/install -r docker -s queue-worker"
```

Comment:
If we remove the queue-worker from here, it would depend on async Jenkins process to process the queue messages.
In CI build, we are running acceptance tests right after init is finished, so there we don't have any way to check if Jenkins has finished the queue messages.
That's why we are manually processing all messages here using queue-worker command

Reply:
The `queue-flush` on line 6 is already clearing any messages that were in the queues that could be processed, based on https://github.com/inviqa/harness-base-php/blob/feature/speed-up-spryker-build/src/spryker/application/skeleton/config/install/docker.yml.twig#L90-L105

I might be missing something though

Reply:
If running through the installer (so without a database dump), the demodata will populate several thousand messages, then process them in the next step: https://github.com/inviqa/harness-base-php/blob/feature/speed-up-spryker-build/src/spryker/application/skeleton/config/install/docker.yml.twig#L217-L234

Reply:
Hmm, seems like the install/init steps are changed since I configured them at the beginning, we are now using dedicated install/init bash files instead of their twig versions.

You are right about your last comment, its running the queue-worker from install recipe, so no need to run it from init.sh. However, I'm wondering if we need queue-flush here as well.

Reply:
Removed :)